shell/shared/db.js

```diff
@@ -60,6 +60,8 @@ UserActions = new Mongo.Collection("userActions");
 //   userId:  User who has installed this action.
 //   packageId:  Package used to run this action.
 //   appId:  Same as Packages.findOne(packageId).appId; denormalized for searchability.
+//   appName:  Same as Packages.findOne(packageId).manifest.appName.defaultText; denormalized for
```

Comment:
"denormalized for searchability" seems incorrect; we don't query on this field.

I guess Mongo doesn't really do joins otherwise I'd suggest we actually don't denormalize this. :)